RocketChat/Rocket.Chat.ReactNative
# [FIX] Showing blank screen on share extension if the user is not logged in

## Proposed changes

Redirect users to WithoutServersView in case there was no logged in server.

## Issue(s)	


## How to test or reproduce

- Make sure the app is not logged in any server (if you open from app icon, shows onboarding screen)
- Open Photos or Gallery and try to share a photo using [Rocket.Chat](http://rocket.chat/)


## Screenshots


## Types of changes




## Checklist



## Further comments
